update steps for cluster page changes (#11883)
pingcap/docs: tidb-cloud/create-tidb-cluster.md

To create a Dedicated Tier cluster, take the following steps:

1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page.

> 2. If you have multiple projects, choose a target project in the left navigation pane, and then click **Create Cluster** in the upper-right corner. Otherwise, click **Create Cluster** in the upper-right corner directly.

Author: ```suggestion
2. If you have multiple projects, switch to the target project in the left navigation pane, and then click **Create Cluster** in the upper-right corner. Otherwise, click **Create Cluster** in the upper-right corner directly.
```

> 

Author: Do you mean L107?

> 

Reviewer: Yes.
```markdown
4. If this is the first cluster of your current project and CIDR has not been configured for this project, you need to set the project CIDR, and then click **Next**. If you do not see the **Project CIDR** field, it means that CIDR has already been configured for this project.
```

> 

Author: Get it. Updated in https://github.com/pingcap/docs/pull/11883/commits/3e7dff08942d0571483a99786b70e7ae7e3be13b.
